pkg/koordlet/util/system/cgroup_resource.go

```diff
@@ -292,7 +303,9 @@ func (c *CgroupResource) IsSupported(parentDir string) (bool, string) {
 		if c.CheckSupported == nil {
 			return false, "unknown support status"
 		}
-		c.Supported, c.SupportMsg = c.CheckSupported(c, parentDir)
+		newSupported, isSupported, msg := c.CheckSupported(c, parentDir)
```

Comment:
what's the differences of newSupported and isSupported

Reply:
fixed.

now there are two types of CheckSupported:
1. use `WithSupported(bool, string)` to initialize a support status when the resource is created.
2. use `WithCheckSupported(func(Resource, string) (bool, string))` to get the support status every time read or write a file of the resource.